Ginger/GingerTest/ActionConversion/ActionConversionTest.cs

```diff
@@ -991,6 +1004,30 @@ public void JavaActSwitchWindowToUISwitchWindow()
             ActSwitchWindowTOUISwitchWindowConvertor(activity);
         }
 
+        [TestMethod]
+        [Timeout(100000)]
+        public void JavaLegacyPOMConversionTest()
+        {
+            //Arrange
+            NewRepositorySerializer RepositorySerializer = new NewRepositorySerializer();
+            var businessFlowFile = TestResources.GetTestResourcesFile(@"JavaLegacyToPOMxml" + Path.DirectorySeparatorChar + "Java_Legacy_Actions_BF.Ginger.BusinessFlow.xml");
+            var javaPomFile = TestResources.GetTestResourcesFile(@"JavaLegacyToPOMxml\Java Swing Test App.Ginger.ApplicationPOMModel.xml");
+            
+            //Load BF
+            mBF = (BusinessFlow)RepositorySerializer.DeserializeFromFile(businessFlowFile);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4824688)